### PR TITLE
When a PR goes from auto mergable to not - remove the comment offering to merge the PR

### PR DESCRIPTION
**Pull request by orta**
We just had a big PR which made a lot of PRs get merge conflicts, and a few have now gone from "ok, I can merge this for you" back to "this needs a rebase, then reviews" - we need to kill the message saying it can merge.

### CODE REVIEW

File: src/compute-pr-actions.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export interface Actions {
     shouldUpdateLabels: boolean;
     shouldUpdateProjectColumn: boolean;
     shouldRemoveFromActiveColumns: boolean;
+    isClassedAsReadyForMerge: boolean
```
Comment (sandersn) [inline src/compute-pr-actions.ts:43]: I assume prettier is going to add a semicolon here to match the other lines (or perhaps...remove the others?)

File: src/compute-pr-actions.ts [hunk 2]
```diff
@@ -121,6 +124,7 @@ export function process(info: PrInfo | BotEnsureRemovedFromProject | BotNoPackag
     context.labels["New Definition"] = info.anyPackageIsNew;
     context.labels["Author is Owner"] = info.authorIsOwner;
     context.labels["Merge:Auto"] = canBeMergedNow(info);
+    context.isClassedAsReadyForMerge = canBeMergedNow(info);
```
Comment (sandersn) [inline src/compute-pr-actions.ts:127]: so the problem was that `canBeMergedNow` wasn't persisted before?

File: src/compute-pr-actions.ts (hunk 1 above)
Comment (andrewbranch) [reply src/compute-pr-actions.ts:43]:
```suggestion
    isMergeable: boolean;
```

?

“classed as” feels very clunky

**Comment by orta** [comment]
@andrewbranch how dare you defy up my old school cocoa naming structure

That said, maybe "isReadyForAutoMerge" is probably the best (is mergeable could be that the PR is in a mergeable state)

File: src/compute-pr-actions.ts (hunk 1 above)
Comment (orta) [reply src/compute-pr-actions.ts:43]: Nah, I've not set up prettier for this repo - some of the older code would probably look pretty bad so I didn't add it after I took over

File: src/compute-pr-actions.ts (hunk 2 above)
Comment (orta) [reply src/compute-pr-actions.ts:127]: effectively it wasn't being passed down to the 'commenting' stage and couldn't be used to show/remove that message

**Pull request merged**